### DISCUSSION TITLE
Unable to access Applications settings

### DISCUSSION THREAD

**Discussion by bago2k4** [airbytehq/airbyte#39528] [Connector Questions] [answered]
Hi,
I have Airbyte running via Docker. I need to create an api key and secret to connect to it via REST api.
When I open the settings panel I don't see the Applications link in the side panel, but only Profile is visible under the Account group.

Is there something I can do about it? is my configuration wrong or it's a bug?

<img width="218" alt="Screenshot 2024-06-17 at 19 05 15" src="https://github.com/airbytehq/airbyte/assets/642704/96aeecf1-ab7a-420c-8900-f9b84304d451">

**Comment by jkleinkauff** [comment]
Same here. We want to be able to use the API thus it seems we need the settings/applications page. Can someone help?

**Reply by verbotenj** [reply]
Auth has to be enabled to see Applications under Account.

**Comment by bago2k4** [comment, marked as answer]
@jkleinkauff I found out that APIs are accessible using basic http authentication instead of using the API keys.

Settings/Applications is what they call "API Key Management" in the pricing page and that's not included in the free slice.

Hope this helps you or others in the future
